Dockerfile

```diff
@@ -124,6 +124,7 @@ ENV PATH="/usr/local/miniconda/bin:$PATH" \
     LC_ALL="C.UTF-8" \
     PYTHONNOUSERSITE=1
 
+ENV FOO=1
```

Comment:
This one in particular mystifies me. But if it's not there, I get a failure to import matplotlib a couple of steps down (in the font caching step).

Reply:
What is the exact error?

Reply:
(I'm going to merge because this is not really relevant)

Reply:
I get an import error when importing matplotlib on https://github.com/nipreps/dmriprep/blob/master/Dockerfile#L154, as though the installation on https://github.com/nipreps/dmriprep/blob/master/Dockerfile#L129 didn't work.